authz/authz-cr-validation/run-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit;
+set -o nounset;
+set -o pipefail;
+set -o xtrace;
+
+check_cr_validation() {
+    path="$1"
+    fail="$2"
+    crd_path="./gateway-api/config/mesh/crd/experimental" 
+    set +e
+    output=$(kubectl validate "${path}" --local-crds "${crd_path}"  2>&1)
```

Comment:
Where are you installing this tool? Would be great to have something that installed kubectl-validate if it wasn't already present

Reply:
done!!